tsig.go

```diff
@@ -166,10 +166,12 @@ type timerWireFmt struct {
 // timersOnly is false.
 // If something goes wrong an error is returned, otherwise it is nil.
 func TsigGenerate(m *Msg, secret, requestMAC string, timersOnly bool) ([]byte, string, error) {
```

Comment:
on the docs - it's weirdly formatted, also Tsig RR -> TSIG RR..

"... called for the first time requestMAC should be set to the empty string and timersOnly should be false.":

would also be nice to tell what subsequent calls should look like,.

The "if something goes wrong an error is returned..." can be removed IMO

Reply:
I have made no change in the documentation for that function.